client/my-sites/plans/current-plan/ecommerce-trial/index.tsx

```diff
@@ -0,0 +1,266 @@
+		},
+		{
+			title: translate( 'Sell your products' ),
+			text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis quis magna ac odio ullamcorper efficitur.',
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/money.png',
+		},
+		{
+			title: translate( 'Connect with your customers' ),
+			text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis quis magna ac odio ullamcorper efficitur.',
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/email.png',
+		},
+		{
+			title: translate( 'Promote your products' ),
+			text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis quis magna ac odio ullamcorper efficitur.',
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/promote.png',
+		},
+		{
+			title: translate( 'Integrate top shipping carriers' ),
+			text: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis quis magna ac odio ullamcorper efficitur.',
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/shipping.png',
+		},
+	];
+
+	const trialStart = moment( currentPlan?.subscribedDate );
+	const trialEnd = moment( currentPlan?.expiryDate );
+	const trialDuration = trialEnd.diff( trialStart, 'days' );
+
+	/**
+	 * Trial progress from 0 to 1
+	 */
+	const trialProgress = 1 - eCommerceTrialDaysLeft / trialDuration;
+	const eCommerceTrialDaysLeftToDisplay = isTrialExpired ? 0 : eCommerceTrialDaysLeft;
+
+	// moment.js doesn't have a format option to display the long form in a localized way without the year
+	// https://github.com/moment/moment/issues/3341
+	const readableExpirationDate = eCommerceTrialExpiration?.toDate().toLocaleDateString( locale, {
+		month: 'long',
+		day: 'numeric',
+	} );
+
+	return (
+		<>
+			<BodySectionCssClass bodyClass={ [ 'is-trial-plan' ] } />
+
+			<Card className="e-commerce-trial-current-plan__trial-card">
+				<div className="e-commerce-trial-current-plan__trial-card-content">
+					<p className="e-commerce-trial-current-plan__card-title">
+						{ translate( 'You’re in a free trial' ) }
+					</p>
+					<p className="e-commerce-trial-current-plan__card-subtitle">
+						{ isTrialExpired
+							? translate(
+									'Your free trial has expired. Sign up to a plan to unlock new features and keep your store running.'
+							  )
+							: translate(
+									'Your free trial will end in %(daysLeft)d day. Sign up to a plan by %(expirationdate)s to unlock new features and keep your store running.',
+									'Your free trial will end in %(daysLeft)d days. Sign up to a plan by %(expirationdate)s to unlock new features and keep your store running.',
+									{
+										count: eCommerceTrialDaysLeft,
+										args: {
+											daysLeft: eCommerceTrialDaysLeft,
+											expirationdate: readableExpirationDate,
+										},
+									}
+							  ) }
+					</p>
+					<Button className="e-commerce-trial-current-plan__trial-card-cta" primary>
+						{ translate( 'Get Commerce' ) }
```

Comment:
Let's drop the `translate()` here -- I am 99% sure this isn't the copy we're going to ship.